eng/targets/ReadyToRun.targets

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  
+  <Target Name="_InitializeTargetAssembly">
+    <ItemGroup>
+      <_R2ROptimizeAssembly Include="@(IntermediateAssembly)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_LocateCrossgen2">
```

Comment:
Nit: I think you can merge `_LocateCrossgen2` and `_CalculateCrossgen2Invocations` targets into a single one.